scripts/exregional_run_enkf.sh

```diff
@@ -138,7 +138,7 @@ START_DATE=$(echo "${CDATE}" | sed 's/\([[:digit:]]\{2\}\)$/ \1/')
 YYYYMMDDHH=$(date +%Y%m%d%H -d "${START_DATE}")
 
 vlddate=$CDATE
-
+l_fv3reg_filecombined=.false.
```

Comment:
Will l_fv3reg_filecombined be configurable in config.sh or just hardcoded?

Reply:
I'd like to leave it hardcoded in this script, as the default setting. Some time-consuming extra steps are needed if using combined dynvars and tracer files, which is not what we prefer.

Reply:
The original setting for running EnKF was to use ncks to combine the fv_core.res.tile1.nc and fv_tracer.res.tile1.nc files, run EnKF update and then use ncks to separate the two files. This ncks operation takes a long time for large ensemble size and therefore very inefficient. We suggested Ting to change the EnKF code to read in the original fv_core.res.tile1.nc and fv_tracer.res.tile1.nc files so we can skip ncks. Ting added the changes as an additional option with this additional namelist entry (l_fv3reg_filecombined )

Reply:
Thanks for more information on this.